### ISSUE TITLE
Opening Flow UI

### ISSUE THREAD

**Issue by glarimy** [glarimy/glarimy-quiz-app#27]
Develop the UI that is to be displayed when the app is opened. No java code is included into this issue.

**Comment by glarimy** [comment]
Implement the flow covered in the screen 2/23 of the finalized UI flow referred. It includes coding the MainActivity, layouts and etc., No images to be used. 

Complete this activity by CoB tomorrow.

**Comment by ramakrishna5a1** [comment]
sir from where the activity name to be taken ?
is it from the flow diagram ....?
for  example flow covered in the screen 2 has **ShowWelcomeMessage**

**Comment by glarimy** [comment]
that's right. welcome screen as specified in slide-2 in that 23-slide pdf.

**Comment by glarimy** [comment]
Nothing specific. Checking the code.

**Comment by ramakrishna5a1** [comment]
ok sir

**Comment by glarimy** [comment]
Any update?

**Comment by ramakrishna5a1** [comment]
i am checkin the code sir

**Comment by glarimy** [comment]
Thanks!

**Comment by glarimy** [comment]
The is built using the code based at 2PM today. When running the app on the device, the screen doesn't look like how it should be. Nothing is in sync with the screen that is being expected?

Answer the following questions, one by one.
1. Have you built the app after coding?
2. Have you seen the welcome screen appearing on the device?
3. The text that appeared on the screen was 'welcome'?
4. Was the screen background color black?
5. The logo 'Glarimy Quiz' was appearing on the bottom of the screen?
6. The text 'welcome' was in strawberry color?

Need the answers in the next 30 minutes.

**Comment by ramakrishna5a1** [comment]
sir i  am correcting all of the problems, some of the resource values are disturbed for that reason the layouts are disturbed.

i am correcting these problems,i will update the status soon

and now i  am include the 'Glarimy Quiz' at the bottom of the layout

**Comment by glarimy** [comment]
Sorry ... not acceptable! I need answer for each of the questions that I have posted, ASAP.

**Comment by ramakrishna5a1** [comment]
1,sir i am builted the app after code  coding

2,yes sir i am seen the welcome message on the screen off my device.

3,text appeared is welcome

4,screen background color is black

5,the logo was appearing at the bottom of the screen

6,The text  appeared 'welcome' is in strawberry color



sir i am checked in the code now after resolving the problems

**Comment by ramakrishna5a1** [comment]
![screenshot_2018-05-17-16-13-49-933_com glarimy quiz app 1](https://user-images.githubusercontent.com/32598018/40172858-86f39f1e-59ed-11e8-8135-053126a9870d.png)
